apps/hello-world/next.config.js

```diff
@@ -1,22 +1,56 @@
-//@ts-check
-
-// eslint-disable-next-line @typescript-eslint/no-var-requires
+const path = require('path')
 const { composePlugins, withNx } = require('@nx/next')
+const { createVanillaExtractPlugin } = require('@vanilla-extract/next-plugin')
+const withVanillaExtract = createVanillaExtractPlugin()
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer')
 
 /**
  * @type {import('@nx/next/plugins/with-nx').WithNxOptions}
  **/
 const nextConfig = {
-  nx: {
-    // Set this to true if you would like to use SVGR
-    // See: https://github.com/gregberge/svgr
-    svgr: false,
+  webpack: (config, { isServer }) => {
+    config.resolve.alias.canvas = false
+
+    if (process.env.ANALYZE === 'true' && !isServer) {
+      config.plugins.push(
+        new BundleAnalyzerPlugin({
+          analyzerMode: 'static',
+          reportFilename: isServer
+            ? '../analyze/server.html'
+            : './analyze/client.html',
+        }),
+      )
+    }
+
+    const modules = path.resolve(__dirname, '../..', 'node_modules')
+
+    config.resolve.alias = {
+      ...(config.resolve.alias || {}),
+      '@babel/runtime': path.resolve(modules, '@babel/runtime'),
+      'bn.js': path.resolve(modules, 'bn.js'),
+      'date-fns': path.resolve(modules, 'date-fns'),
+      'es-abstract': path.resolve(modules, 'es-abstract'),
+      'escape-string-regexp': path.resolve(modules, 'escape-string-regexp'),
+      'readable-stream': path.resolve(modules, 'readable-stream'),
+      'react-popper': path.resolve(modules, 'react-popper'),
+      inherits: path.resolve(modules, 'inherits'),
+      'graphql-tag': path.resolve(modules, 'graphql-tag'),
+      'safe-buffer': path.resolve(modules, 'safe-buffer'),
+      scheduler: path.resolve(modules, 'scheduler'),
+    }
+
+    return config
+  },
+
+  env: {
+    API_MOCKS: process.env.API_MOCKS || '',
```

Comment:
You might not need all of this.

Reply:
I tried removing all of it and it builds successfully, maybe in the future there might be a need for this.